src/themes/hugo-theme-learn/static/js/learn.js

```diff
@@ -179,7 +179,7 @@ jQuery(document).ready(function() {
         var code = $(this),
             text = code.text();
 
-        if (text.length > 5) {
+        if (text.length > 80) { // We have a minimum limit to avoid the copy-to-clipboard on `small` `tags`
```

Comment:
for 20-10 it was already triggering for short ones in this page which was weird, May be we should differentiate between single-line and multi-line?

Reply:
It also influences that I hate that copy clipboard thing, is rather useless and annoying, but it's just my opinion :P :) , I will shorten it to 30 for now.

Reply:
I don't know I think it might be useful for longer commands since you dont have to fight the mouse to copy it :)

I don't mind turning it off for short commands (with the single back-ticks) and leaving it on for the code blocks (triple back-ticks) buuut IDK if it's easily possible :)